src/LiteCore/src/LiteCore.Shared/Interop/C4Replicator.cs

```diff
@@ -61,9 +56,9 @@ internal sealed class ReplicatorParameters : IDisposable
         private C4ReplicatorParameters _c4Params;
         private C4ReplicatorBlobProgressCallback _onBlobProgressUpdated;
         private C4ReplicatorStatusChangedCallback _onStatusChanged;
-        private C4ReplicatorDocumentEndedCallback _onDocumentEnded;
-        private C4ReplicatorPushFilterFunction _pushFilter;
-        private C4ReplicatorValidationFunction _validation;
+        private C4ReplicatorDocumentErrorCallback _onDocumentEnded;
```

Comment:
Why does this keep changing back and forth?

Reply:
I modified C4ReplicatorDocumentErrorCallback to C4ReplicatorDocumentEndedCallback in feature replication eventing. But this PR should not have that change yet...so I changed it back..